src/Zigzag_persistence/doc/Intro_zigzag_persistence.h

```diff
@@ -0,0 +1,71 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+ *    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+ *    Author(s):       Hannah Schreiber
+ *
+ *    Copyright (C) 2023 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+#ifndef DOC_ZIGZAG_PERSISTENCE_INTRO_ZIGZAG_PERSISTENCE_H_
+#define DOC_ZIGZAG_PERSISTENCE_INTRO_ZIGZAG_PERSISTENCE_H_
+
+// needs namespace for Doxygen to link on classes
+namespace Gudhi {
+namespace zigzag_persistence {
+
+/** \defgroup zigzag_persistence Zigzag Persistence
+ * @{
+ * \author    Cl&eacute;ment Maria, Hannah Schreiber
+ *
+ * \section zigzagintro Zigzag Persistence
+ *
+ * We refer to the introduction page \ref persistent_cohomology for persistent (co)homology for an introduction
+ * to the topic.
+ * Zigzag persistence is a generalization of the latter. While standard persistence only allows to grow the filtered
+ * complex by adding faces, zigzag persistence also allows removals. Hence the name "zigzag", as the module
+ * diagram will have arrows alternating between forward and backward.
+ *
+ * The module is partitioned in two types of classes: filtered by filtration values and filtered by the atomic
+ * operations.
```

Comment:
They are, but the user don't see it.

Reply:
Ok, they see it because I force them to decide about the elementary order, so I see what you mean. But in the end result, the barcode will not change if the operations with same filtration values are not done in the same order as long as the face/coface order is maintained (ie the order is a valid filtration). The difference can only be seen with the other version.

Reply:
That sounds wrong.
![zz](https://github.com/GUDHI/gudhi-devel/assets/10097863/5cdb0f33-150f-46e9-bc95-c217e081ecb0)
(T is the triangle)
If you start with the black part and e (at filtration -1), then do +f -f +f +T -T -e (in this order, all at the same filtration value 0), and keep it identical (black+green) until filtration 1, you have a long bar [-1,1] corresponding to a 1-cycle that persists through the whole thing. If you do the same operations in the order +f +T -T -e -f +f, there is a step where no 1-cycle exists, so the bar is split into [-1,0] and [0,1]. The local order doesn't only have local effect, it can also influence what happens to long-lived features.

Reply:
I tried again to reformulate this paragraph and added the notion of wrapping (it's a more technical term, but probably the best way to describe it). What do you think?

Reply:
ok (I am not very fond of using the word "index" for the filtration values, but there is enough context that it should be fine)

I notice that you don't mention anymore that cells can be represented by arbitrary faceID in the wrappers. That's ok, but then I think it would make sense to have a paragraph in the doc of the wrappers talking about faceID, or users might be surprised by this difference between the base class and the wrappers.

Reply:
> I notice that you don't mention anymore that cells can be represented by arbitrary faceID in the wrappers. That's ok, but then I think it would make sense to have a paragraph in the doc of the wrappers talking about faceID, or users might be surprised by this difference between the base class and the wrappers.

The sentence is still there, though? ("the boundaries do not have to be ordered, nor identified continuously from 0") It was the same before.

Reply:
:eyeglasses: Uh, how did I miss it? It is indeed still there.